proto/money/folio/folio_state.proto

```diff
@@ -19,5 +20,10 @@ message FolioState {
 	.holms.types.primitive.MonetaryAmount gross_refunds = 6;
 	.holms.types.primitive.MonetaryAmount available_payment_authorization_balance = 7;
 	.holms.types.primitive.MonetaryAmount available_guarantee_authorization_balance = 8;
+
+	.holms.types.booking.reservations.ReservationSummary master_folio = 9;
+	repeated .holms.types.booking.reservations.ReservationSummary client_folios = 10;
```

Comment:
This doesn't belong here.

I know I'm dumping a lot of work onto you by saying this, but you really need to take this off of here. It's not even in the same context and requires materializing a ton of state that's just as easily done from the client.